### PR TITLE
STAR-577 add disabled option to the context menu

### PR DESCRIPTION
**Pull request by ImreBognarUltimaker**
STAR-577
Add a disabled option to the context menu component.

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -29,7 +29,27 @@
     semver "^5.4.1"
     source-map "^0.5.0"
 
-"@babel/core@^7.1.0", "@babel/core@^7.1.6", "@babel/core@^7.2.2":
+"@babel/core@^7.1.0":
```
Comment (ChrisTerBeke) [inline yarn.lock]: Why did so many dependencies change? A diff of 1500+ lines.

Comment (ImreBognarUltimaker) [reply yarn.lock]: This branch is based on this PR:
https://github.com/Ultimaker/react-web-components/pull/203

Large libraries were updated.

Comment (ChrisTerBeke) [reply yarn.lock]: It's hard to see when basing branches on other branches than master. It's better to leave those completely separate PRs (as you cannot merge this branch now until the other one is unless you accept all those updates as well).

Comment (ImreBognarUltimaker) [reply yarn.lock]: I will make PRs like this a draft in the future, until the one on which it's based on is merged.

**Pull request merged**